Update references.md (#117)
shellspec/shellspec: docs/references.md

This is primarily designed for external command calls.
The external command does not have to be a shell script.

> Even shell scripts are processed as external commands (respect shebang), so it is not covered by coverage.

Author: ```suggestion
Even shell scripts are processed as external commands (and thus respecting shebang), so they are not covered by coverage.
```

Reviewer: Thank you for your suggestions and sorry for the confusion.

Is the meaning conveyed with this sentence?

> Even shell scripts are executed as external commands according to the shebang, so they are not covered by coverage.

Author: Yes, I think it is better. 
Thank you!

> Even shell scripts are processed as external commands (respect shebang), so it is not covered by coverage.

Author: ```suggestion
Even shell scripts are executed as external commands according to the shebang, so they are not covered by coverage.
```

Author: (I added it if you want, or just close this PR, as you wish ;))
